test/parallel/test-buffer-isbuffer.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+require('../common');
+const assert = require('assert');
+const { Buffer } = require('buffer');
```

Comment:
> Our linter forbids using Buffer via the global

I think that's only true in `lib`. Tests can use the `Buffer` global and the linter won't complain.